formatTest/typeCheckedTests/input/attributes.re

```diff
@@ -250,3 +250,12 @@ let myFun = fun ((X hello | Y hello) [@onOrPattern]) => hello;
 /* Bucklescript FFI item attributes */
 
 external imul : int => int => int = "Math.imul" [@@bs.val];
+
+let module Js = {
+  type t 'a;
+};
+
+type classAttributesOnKeys = <
+  key1 [@bs.set] : string,
+  key2 [@bs.get {null}] : Js.t int [@onType2]
```

Comment:
So `[@onType2]` applies to `Js.t int`.

How do we apply an attribute *only* to `int`? I think there should be a test case for it.

Also, as I suggested in the discussion, I think we should also be able to do `(Js.t int [@onType2])`.